tests/components/advantage_air/test_config_flow.py

```diff
@@ -0,0 +1,71 @@
+"""Test the Advantage Air config flow."""
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.advantage_air.const import DOMAIN
+
+from tests.async_mock import patch
+from tests.components.advantage_air import TEST_SYSTEM_DATA, TEST_SYSTEM_URL, USER_INPUT
+
+
+async def test_form(hass, aioclient_mock):
+    """Test that form shows up."""
+
+    aioclient_mock.get(
+        TEST_SYSTEM_URL,
+        text=TEST_SYSTEM_DATA,
+    )
+
+    result1 = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result1["type"] == data_entry_flow.RESULT_TYPE_FORM
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6319846)